SDL/Quintuplet.h

```diff
@@ -3073,7 +3075,7 @@ namespace SDL {
                       mdsInGPU.anchorEta[segmentsInGPU.mdIndices[2 * tripletsInGPU.segmentIndices[2 * innerTripletIndex +
                                                                                                   layer2_adjustment]]];
                   float pt = (innerRadius + outerRadius) * 3.8f * 1.602f / (2 * 100 * 5.39f);
-                  float scores = chiSquared + nonAnchorChiSquared;
+                  float scores = inference;
```

Comment:
I forgot that the non-anchor hits are used in the scoring.
Abandoning it would be a loss of information.
This gives me a new idea though: to add radius computation for non-anchor hits and combine with the anchor data in the inference.